crypto/fipsmodule/rand/getrandom_fillin.h

```diff
@@ -34,6 +34,8 @@
 #define EXPECTED_NR_getrandom 359
 #elif defined(OPENSSL_RISCV64)
 #define EXPECTED_NR_getrandom 278
+#elif defined(OPENSSL_S390X)
+#define EXPECTED_NR_getrandom 349
```

Comment:
For future reference, this value aligns [with `getrandom` in the the syscall table](https://github.com/torvalds/linux/blob/99bd3cb0d12e85d5114425353552121ec8f93adc/arch/s390/kernel/syscalls/syscall.tbl#L359).